specs/~dynamic-partials.yml

```diff
@@ -0,0 +1,140 @@
+overview: |
+  Dynamic partials tags are used to dynamically expand an external template into
+  the current template.
+
+  The tag's content MUST be a non-whitespace character sequence NOT containing
+  the current closing delimiter.
+
+  This tag's content names a key in the context whose value is the name of the
+  partial that will be loaded. If the dynamically named partial cannot be found,
+  the empty string SHOULD be used instead, as in interpolations. Set Delimiter
+  tags MUST NOT affect the parsing of a partial.  The partial MUST be  rendered
+  against the context stack local to the tag. Failed resolutions of the key
+  (context lookups) should be considered falsey and should interpolate as the
+  empty string. If the partial, whose name is retrieved from the context stack,
+  cannot be found, the empty string SHOULD be used instead, as in
+  interpolations.
+
+  Dynamic partial tags SHOULD be treated as standalone when appropriate.  If
+  this tag is used standalone, any whitespace preceding the tag should treated
+  as indentation, and prepended to each line of the partial before rendering:
+  whitespace handling around dynamic partials SHOULD be identical to whitespace
+  handling around static (normal) partials.
+tests:
+  - name: Basic Behavior
+    desc: The asterisk operator is used for dynamic partials.
+    data: { dynamic: 'content' }
+    template: '"{{*dynamic}}"'
+    partials: { content: 'Hello, world!' }
+    expected: '"Hello, world!"'
+
+  - name: Context Misses
+    desc: Failed context lookups should be considered falsey.
+    data: { }
+    template: '"{{*missing}}"'
+    partials: { }
+    expected: '""'
+
+  - name: Context Misses Again
+    desc: Failed context lookups should be considered falsey.
+    data: { }
+    template: '"{{*missing}}"'
+    partials: { missing: 'Hello, world!' }
+    expected: '""'
+
+  - name: Failed Lookup
+    desc: The empty string should be used when the named partial is not found.
+    data: { dynamic: 'content' }
+    template: '"{{*dynamic}}"'
+    partials: { foobar: 'Hello, world!' }
+    expected: '""'
+
+  - name: Context
+    desc: The asterisk operator should operate within the current context.
+    data: { text: 'Hello, world!', example: 'partial'  }
+    template: '"{{*example}}"'
+    partials: { partial: '*{{text}}*' }
+    expected: '"*Hello, world!*"'
+
+  - name: Recursion
+    desc: The asterisk operator should properly recurse.
+    data: |
```

Comment:
the | on line 61 is causing rest of data to be interpreted as a single string
merging 61 and 62 (removing the |) fixes the issue